packages/replay/package.json

```diff
@@ -36,10 +36,11 @@
   "dependencies": {
     "@replayio/sourcemap-upload": "workspace:^",
     "@types/semver": "^7.5.6",
-    "commander": "^7.2.0",
+    "commander": "^12.0.0",
```

Comment:
just an observation: this changes the minimal required node version quite a bit (it's now node 18 whereas previously it was probably smth like node 10). Other dependencies might already require newer versions than node 10 though ;p and in general, I think it's probably just fine for this CLI here